README.md

```diff
@@ -53,8 +50,7 @@ To contribute a patch, follow these instructions (note that installing
 ```
 
 
-License (see also LICENSE.txt)
-==============================
+## License (see also LICENSE.txt)
```

Comment:
great changes, these are orthogonal though - can you separate them into a new PR apart from this?